tern/__main__.py

```diff
@@ -153,6 +158,18 @@ def main():
     parser_report.add_argument('-w', '--raw-image', metavar='FILE',
                                help="Raw container image that exists locally "
                                "in the form of a tar archive.")
+    parser_report.add_argument('-l', '--layer', metavar='LAYER_NUMBER',
+                               type=int, default=0,
+                               help="Layer number of the image to analyze."
+                               " Base OS layer is 1. Can only be used with"
+                               " -i --docker-image analysis. Default to 0"
+                               " which means the feature is turned off")
```

Comment:
It seems counter intuitive to have the default value be to turn off this feature (maybe this is just me though?). If the user goes through the trouble of writing `--layer` on the command line, I would think more times than not they are interested in it on some level. What do you think about the default value being a report for just the base layer?

Reply:
makes sense :)